pkg/sbom/spdx/unmarshal_test.go

```diff
@@ -66,6 +66,7 @@ func TestUnmarshaler_Unmarshal(t *testing.T) {
 											},
 										},
 									},
+									BOMRef: "Package-b7ebaf0233f1ef7b",
```

Comment:
`github.com/spdx/tools-golang` returns `Package SPDX identifier` without `SPDXRef-` prefix.
On the one hand, docs say: `"SPDXRef-"[idstring] where [idstring] is a unique string containing letters, numbers, ., and/or -. `(https://spdx.github.io/spdx-spec/v2.3/package-information/#72-package-spdx-identifier-field). So id is text after `SPDXRef-`.
On the other hand, `BOMRef` will not match string from `SPDXID` field.